constraints.go

```diff
@@ -345,8 +343,47 @@ func constraintNotEqual(v *Version, c *constraint) (bool, error) {
 	return true, nil
 }
 
-func constraintGreaterThan(v *Version, c *constraint) (bool, error) {
+// Constraint functions
+func constraintEqual(v *Version, c *constraint) (bool, error) {
```

Comment:
could have simply used the constraintNotEqual and return the opposite result on error (except for the pre-release check)